src/js/lib/client/src/index.js

```diff
@@ -177,6 +176,15 @@ function parseCmdArgs() {
 
 let args = parseCmdArgs()
 
+// Note: this is a conditional default to avoid issues with some window managers affecting
+// interactions at the top of a borderless window. Thus, we want borders on Win/Linux and
+// borderless on Mac. See https://github.com/enso-org/ide/issues/1101 for details.
+if (args.frame === undefined) {
+    const show_border = process.platform !== 'darwin'
+    args.frame = show_border
+}
```

Comment:
Why do we use so many names for the same thing? If we are using name `frame`, lets stick to it, instead of naming it "border" as well.